packages/manager/modules/vps/src/upscale/upscale.controller.js

```diff
@@ -479,8 +489,10 @@ export default class UpscaleController {
 
     this.loading.performUpscale = true;
     const planCode = this.planCode || this.range.planCode;
+    const currentPlanCode = this.upscaleRanges.find((e) => e.isCurrentRange)
+      .planCode;
 
-    return this.performUpscale(planCode)
+    return this.performUpscale(planCode, currentPlanCode)
```

Comment:
**question:**  Why is this change made. I dont see its usage.

Reply:
it's use on

.then(() => {
        this.atInternet.trackPage({
          name: `upscale-confirm-banner::${currentPlanCode}_to_${planCode}::success`,
          type: 'navigation',
        });
      })
      .catch((error) => {
        this.atInternet.trackPage({
          name: `upscale-confirm-banner::${currentPlanCode}_to_${planCode}::error`,
          type: 'navigation',
        });

Reply:
But you dont need to pass it to `performUpscale` function.